h2o-genmodel/src/main/java/hex/genmodel/tools/PredictCsv.java

```diff
@@ -182,6 +182,8 @@ public void run() throws Exception {
         RowData row = formatDataRow(splitLine, inputColumnNames);
         // Do the prediction.
         // Emit the result to the output file.
+        String offsetColumn = modelWrapper.m.getOffsetName();
+        double offset = offsetColumn == null ? 0 : Double.valueOf((String) row.get(offsetColumn));
```

Comment:
it is better to use Double.parseDouble because you avoid unnecessary creation of Double object

Reply:
Nice.